src/Form/AdminSettingsForm.php

```diff
@@ -381,7 +447,7 @@ private function setCredentialValues(Config $settings, array $values) {
     }
     $settings->clear('credential.oauth_url');
     if (!empty($values['oauth_url'])) {
-      $settings->set('credential.oauth_url', 'https://' . $this->cleanUrl($values['oauth_url']));
+      $settings->set('credential.oauth_url', 'https://' . $this->cleanUrl($this->removeAuthorizeSuffix($values['oauth_url'])) . '/authorize');
```

Comment:
When saving to DB, the "/authorize" is added BEFORE saving. This means the value in DB stays the same and there is no migration required.